examples/just-teams/main.tf

```diff
@@ -0,0 +1,18 @@
+# Pre-populate parameter store with your Teams webhook URL
+data "aws_ssm_parameter" "sns_topic_teams_webhook" {
+  name = "/teams/webhooks/test_url"
+}
+module "sns" {
+  #source  = "USSBA/sns-notification-framework/aws"
+  #version = "~> 2.1"
```

Comment:
Might as well bump this source version to 3.0 -- could see people (e.g. me) doing a copy paste and messing this up. Also teams doesn't work in 2.x so there is that too.